### PR TITLE
Shubham dalal work

### PR DESCRIPTION
**Pull request by shubhamdalal49**

### CODE REVIEW

File: Shubham_Dalal/JavaScript/Assignment4/ams.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    font-size: 20px;
+    color: white;
+    background-image: radial-gradient(black, black, black, rgb(7, 94, 71), white, rgb(7, 94, 71));
+}
+h2{
+    text-align: center;
+}
+h3{
+    text-align: center;
+}
```
Comment (Pawan1005) [inline Shubham_Dalal/JavaScript/Assignment4/ams.css]: add new line

File: Shubham_Dalal/JavaScript/Assignment4/KrishnamurthyNumber.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                    while (temp > 0) {
+                        factorial = 1;
+                        rem = temp % 10;
+                        temp = parseInt(temp / 10);
+                        for (var i = rem; i >= 1; i--) {
+                            factorial = factorial * i;
+                        }
+                        sum = sum + factorial;
+                    }
+
```
Comment (Pawan1005) [inline Shubham_Dalal/JavaScript/Assignment4/KrishnamurthyNumber.html]: remove extra line

File: Shubham_Dalal/JavaScript/Assignment4/AutomorphicNumber.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            <h2>Automorphic Number (*_*)</h2>
+            <input type="number" name="" id="first">
+            <button onclick="automorphic()">Click here</button>
+            <h3 id="square"></h3>
+            <h2 id="result"></h2>
+
+        </div>
+
+    </body>
+</html>
```
Comment (Pawan1005) [inline Shubham_Dalal/JavaScript/Assignment4/AutomorphicNumber.html]: add new line 
remove extra lines

File: Shubham_Dalal/JavaScript/Assignment3/Even_odd.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    </script>
+</head>
+<body>
+    <div>
+        <input type="number" name="" id="num">
+    <button onclick="fun()">Check</button>
+    <h2 id="hid"></h2>
+    </div>
+</body>
+</html>
```
Comment (Pawan1005) [inline Shubham_Dalal/JavaScript/Assignment3/Even_odd.html]: wrong indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Ashwin2812 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Pull request merged**